packages/opentelemetry-core/src/baggage/utils.ts

```diff
@@ -13,33 +13,34 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { Baggage, baggageEntryMetadataFromString } from '@opentelemetry/api';
+import { Baggage, BaggageEntryMetadata, baggageEntryMetadataFromString } from '@opentelemetry/api';
 import {
   BAGGAGE_ITEMS_SEPARATOR,
   BAGGAGE_PROPERTIES_SEPARATOR,
   BAGGAGE_KEY_PAIR_SEPARATOR,
   BAGGAGE_MAX_TOTAL_LENGTH,
 } from './constants';
 
-export const serializeKeyPairs = (keyPairs: string[]) => {
+type ParsedBaggageKeyValue = { key: string, value: string, metadata: BaggageEntryMetadata | undefined };
```

Comment:
Is it possible to use `BaggageEntry` from API? https://github.com/open-telemetry/opentelemetry-js-api/blob/main/src/baggage/types.ts#L35

Reply:
You mean something like this?

``` Typescript
interface ParsedBaggageKeyValue extends BaggageEntry {
  key: string;
}
```

Reply:
hmm I didn't immediately notice the difference. it's fine as-is